ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -139,48 +144,62 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
+        didSet {
+            applyStyle()
+        }
+    }
+
+    private lazy var alignmentConstraint: NSLayoutConstraint = centerXAnchor.constraint(equalTo: containingStackView.centerXAnchor)
+    private var alignment: Alignment = .center {
+        didSet {
+            guard alignment != oldValue else {
+                return
+            }
+            alignmentConstraint.isActive = false
+            let keyPath = alignment.xAxisKeyPath
+            alignmentConstraint = self[keyPath: keyPath].constraint(equalTo: containingStackView[keyPath: keyPath])
+            alignmentConstraint.isActive = true
+        }
+    }
     private var interactivePart: InteractivePart = .none
     private var animatesWhenPressed: Bool = true
     private var accessoryType: AccessoryType = .none
 
-    private let titleButton = EasyTapButton()
-    private var titleAccessoryType: AccessoryType {
-        return interactivePart.contains(.title) ? accessoryType : .none
-    }
+    private lazy var containingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        return stackView
+    }()
 
-    var currentStyle: Style {
-        didSet {
-            applyStyle()
-        }
+    private static func makeEmptyTitleLineStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.spacing = TokenSetType.titleStackSpacing
+        return stackView
     }
 
-    private lazy var titleButtonLabel: Label = {
+    private lazy var titleContainer = Self.makeEmptyTitleLineStackView()
+    private lazy var subtitleContainer = Self.makeEmptyTitleLineStackView()
+
+    private lazy var titleLabel: Label = {
         let label = Label(textStyle: TokenSetType.defaultTitleFont)
         label.lineBreakMode = .byTruncatingTail
         label.textAlignment = .center
         return label
     }()
 
-    private var titleButtonLeadingImageView = UIImageView()
-    private var titleButtonTrailingImageView = UIImageView()
-
-    private var titleButtonLeadingImageAreaWidth: CGFloat {
-        return titleButtonLeadingImageView.image != nil ? 2 * TokenSetType.leadingImageTotalPadding : 0
-    }
-
-    private let subtitleButton = EasyTapButton()
-    private var subtitleAccessoryType: AccessoryType {
-        return interactivePart.contains(.subtitle) ? accessoryType : .none
-    }
+    private var titleLeadingImageView = UIImageView()
+    private var titleTrailingImageView = UIImageView()
 
-    private lazy var subtitleButtonLabel: Label = {
+    private lazy var subtitleLabel: Label = {
         let label = Label(textStyle: TokenSetType.defaultSubtitleFont)
         label.lineBreakMode = .byTruncatingMiddle
         return label
     }()
 
-    private var subtitleButtonImageView = UIImageView()
+    private var subtitleImageView = UIImageView()
 
     @objc public convenience init(style: Style = .primary) {
```

Comment:
similar how we do it for segmented control, can we write a comment on the the overall view heirarchy which will make it easier for others to follow the code?

Reply:
Added with f14a5e8.